packages/cli/src/deployment.ts

```diff
@@ -183,7 +183,8 @@ async function needToRetry(
   }
   const currentTokens = tokens ? tokens : {}
   const previousTokens = previous.tokens ? previous.tokens : {}
-  const sameWithPrevious = attoAlphAmount === previous.attoAlphAmount && recordEqual(currentTokens, previousTokens)
+  const sameWithPrevious =
+    attoAlphAmount?.toString() === previous.attoAlphAmount?.toString() && recordEqual(currentTokens, previousTokens)
```

Comment:
I found a strange bug, even if `attoAlphAmount` and `previous.attoAlphAmount` are equal, it will return false here, the reason is that `typeof attoAlphAmount` is bigint but `typeof previous.attoAlphAmount` is string, I don’t know why.

Reply:
I will check